Csoar 1450 (#3693)
SumoLogic/sumologic-documentation: docs/cloud-soar/index.md

Navigate the evolving threat landscape with our best-in-class security orchestration, automation and response (SOAR) solution to streamline incident response workflows.

:::info

> The articles in this section only apply to organizations having the latest Clouad SOAR SaaS version. If your Cloud SOAR instance URL matches the pattern `*.soar.sumologic.com`, refer to [Legacy Cloud SOAR](/docs/cloud-soar/legacy/). 

Reviewer: ```suggestion
If your Cloud SOAR instance URL matches the pattern `*.soar.sumologic.com`, this means you're using the legacy version. Refer to the [Legacy Cloud SOAR](/docs/cloud-soar/legacy/) section. 
```
